staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1800,11 +1800,20 @@ type CSIPersistentVolumeSource struct {
 	// controllerExpandSecretRef is a reference to the secret object containing
 	// sensitive information to pass to the CSI driver to complete the CSI
 	// ControllerExpandVolume call.
-	// This is an alpha field and requires enabling ExpandCSIVolumes feature gate.
+	// This is an beta field and requires enabling ExpandCSIVolumes feature gate.
 	// This field is optional, and may be empty if no secret is required. If the
 	// secret object contains more than one secret, all secrets are passed.
 	// +optional
 	ControllerExpandSecretRef *SecretReference `json:"controllerExpandSecretRef,omitempty" protobuf:"bytes,9,opt,name=controllerExpandSecretRef"`
+
+	// nodeExpandSecretRef is a reference to the secret object containing
+	// sensitive information to pass to the CSI driver to complete the CSI
+	// NodeExpandVolume call.
+	// This is an alpha field and requires enabling CSINodeExpandSecret feature gate.
+	// This field is optional, and may be nil if no secret is required. If the
```

Comment:
Sorry for this - but we wouldn't use `nil` because this apidoc is intended for JSON users (I should have clarified).  Now that I think you've answered the question of "the field is absent / omitted" I'd suggest this sentence (And all the others in this file, which can be done later) as:

> This field is optional, may be omitted if no secret is required.

Reply:
@smarterclayton sure,  will address this  in a follow up PR . Thanks!

Reply:
done @humblec @smarterclayton

Reply:
Awesome. thanks @zhucan :+1: